zh/tidb-operator-overview.md

```diff
@@ -34,7 +34,7 @@ TiDB Operator 提供了多种方式来部署 Kubernetes 上的 TiDB 集群：
     - 在公有云上部署生产可用的 TiDB 集群并进行后续的运维管理；
 
         - [Deploy TiDB on AWS EKS](deploy-on-aws-eks.md)
-        - [Deploy TiDB on GCP GKE (beta)](deploy-on-gcp-gke.md)
+        - [Deploy TiDB on Google Cloud GKE (beta)](deploy-on-gcp-gke.md)
         - [Deploy TiDB on Azure AKS](deploy-on-azure-aks.md)
         - [Deploy TiDB on Alibaba Cloud ACK](deploy-on-alibaba-cloud.md)
```

Comment:
```suggestion
        - [在 Google Cloud GKE 上部署 TiDB 集群 (beta)](deploy-on-gcp-gke.md)
        - [在 Azure AKS 上部署 TiDB 集群](deploy-on-azure-aks.md)
        - [在阿里云 ACK 上部署 TiDB 集群](deploy-on-alibaba-cloud.md)
```

@csuzhangxc Please confirm whether `deploy-on-gcp-gke.md` is currently in beta. If it is in beta, we need to make the following updates:
- update the title and add a note in https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/deploy-on-gcp-gke
- update the link text when referring to `deploy-on-gcp-gke.md`, such as:

https://github.com/pingcap/docs-tidb-operator/blob/63ee2f97824944cbc80a6cfe6b7ce3704037f293/zh/get-started.md?plain=1#L843-L846

Reply:
I think we can remove `beta` now.